dashboard/db/migrate/20231102210950_create_lti_courses.rb

```diff
@@ -0,0 +1,16 @@
+class CreateLtiCourses < ActiveRecord::Migration[6.1]
+  def change
+    create_table :lti_courses do |t|
+      t.references :lti_integration, index: true, foreign_key: true, null: false, type: :bigint
+      t.references :lti_deployment, index: false, foreign_key: true, null: false, type: :bigint
+      t.string :context_id
+      t.string :course_id
+      t.string :nrps_url
+
+      t.timestamps
+    end
+
+    add_index :lti_courses, [:lti_integration_id, :context_id], name: 'index_on_lti_integration_id_and_context_id'
+    add_index :lti_courses, [:lti_integration_id, :course_id], name: 'index_on_lti_integration_id_and_course_id'
```

Comment:
Since we will create a separate index for `lti_integration_id`, these combinations will have more sense
```suggestion
    add_index :lti_courses, %i[context_id lti_integration_id], name: 'index_on_context_id_and_lti_integration_id'
    add_index :lti_courses, %i[course_id lti_integration_id], name: 'index_on_course_id_and_lti_integration_id'
```

Reply:
Only the first column in the composite B-tree index really matters for the search.
https://www.mysqltutorial.org/mysql-index/mysql-composite-index

Reply:
Only the first column in the composite B-tree index really matters for the search:
https://www.mysqltutorial.org/mysql-index/mysql-composite-index
<img width="588" alt="Screenshot 2023-11-03 at 01 41 27" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/0e589dfa-beca-4f15-bcec-38f2e8df11fb">

Reply:
Only the first column in the composite B-tree index really matters for the search:
https://www.mysqltutorial.org/mysql-index/mysql-composite-index
<img width="570" alt="Screenshot 2023-11-03 at 01 48 58" src="https://github.com/code-dot-org/code-dot-org/assets/11708250/76ecee4e-69e9-42c0-abd1-917402612155">

Reply:
Nice catch! Seems like a great optimization so I'll add this.